src/python/grpcio/tests/qps/src/proto/grpc/testing/.gitignore

```diff
@@ -0,0 +1 @@
+!*_pb2.py
```

Comment:
The _pb2.py ignore is there because we don't want to check in generated files for python. Please revert this change.